SPECS/mlnx-ofa_kernel/mlnx-ofa_kernel.spec

```diff
@@ -70,14 +70,14 @@ Summary:        Infiniband HCA Driver
 Name:           mlnx-ofa_kernel
 # Update OFED version along with version updates
 Version:        5.6
-Release:        1%{?dist}
+Release:        2%{?dist}
 License:        GPLv2
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Base
 URL:            https://www.mellanox.com/
 Source:         https://www.mellanox.com/downloads/ofed/%{name}-%{MLNX_OFED_VERSION}.tgz#/%{name}-%{version}.tgz
-BuildRequires:  kernel-devel
+BuildRequires:  kernel-devel = 5.15.87.1
```

Comment:
If you have them, I'd also add instructions on how to get the compatible version of the kernel for a future update of this package.

Reply:
Process requires special HW at the moment, specifically the BF2